x-pack/plugins/data_usage/public/app/data_usage.tsx

```diff
@@ -44,7 +43,11 @@ export const DataUsage = () => {
 
   const [queryParams, setQueryParams] = useState<UsageMetricsRequestSchemaQueryParams>({
     metricTypes: ['storage_retained', 'ingest_rate'],
-    dataStreams: [],
+    // TODO: Replace with data streams from /data_streams api
+    dataStreams: [
+      '.alerts-ml.anomaly-detection-health.alerts-default',
+      '.alerts-stack.alerts-default',
+    ],
```

Comment:
Q: With respect to UX, should we use all the datastreams from the API here and have them all selected when we land on the page even if the list is very long, say 100 or more?

Reply:
Probably a question for a PM.  Personally, I would say the "top" 10 would suffice.

Reply:
Yeah, that is my concern. Also since we're pulling the datastreams list from another API (`/_metering/stats`) that doesn't allow for paging, I'm not sure how we're going to handle cases when users have a huge list of data streams.

Reply:
At the moment I think the plan is to not handle it and just display them all.  The dropdown will ideally scroll and not go off the page.